packages/wix-ui-tpa/src/components/Image/Image.st.css

```
@@ -1,13 +1,21 @@
 :import {
   -st-from: "../../common/formatters.st";
-  -st-named: color, fallback;
+  -st-named: color, fallback, font;
+}
+
+:import {
+  -st-from: "../Text/Text.st.css";
+  -st-default: TPAText;
 }
 
 /*Defaults*/
 :vars {
   defaultBorderWidth: 0px;
   defaultBorderRadius: 0px;
   defaultImageOpacity: 100%;
+  errorBackgroundColor: color(color-2);
+  errorTextColor: color(color-1);
+  errorFontStyle: "{theme: 'Body-M', size:'14px' lineHeight: '20px'}";
```

I think we can align to `em` unit:
```suggestion
  errorFontStyle: "{theme: 'Body-M', size:'14px' lineHeight: '1.5em'}";
```

It will be calculated to 21px which is pretty close.